### PR TITLE
HW13

### PR DESCRIPTION
**Pull request by Artsapollo**

### CODE REVIEW

File: src/test/java/Lesson12/StringChangerServiceNewTest.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    @Test
+    public void addStartAndEndUpperTest() {
+        when(stringLowerUpperService.toUpper(Mockito.anyString())).thenReturn("END");
+        String expected = "startmiddleEND";
+        String actual = stringChangerService.addStartAndEndUpper("start", "end", "middle");
+        assertEquals(expected, actual);
+    }
+
+
+    @Test
```
Comment (SergeyPasko) [inline src/test/java/Lesson12/StringChangerServiceNewTest.java]: You don't test any new. It isn't good idea copy code from Yuriy

Comment (Artsapollo) [reply src/test/java/Lesson12/StringChangerServiceNewTest.java]: I know)

Comment (SergeyPasko) [reply src/test/java/Lesson12/StringChangerServiceNewTest.java]: about logic of test

Comment (Artsapollo) [reply src/test/java/Lesson12/StringChangerServiceNewTest.java]: Got u

**Pull request merged**